Userland/Libraries/LibC/stdlib.h

```diff
@@ -57,7 +57,8 @@ long long int llabs(long long int);
 double atof(char const*);
 int system(char const* command);
 char* mktemp(char*);
-int mkstemp(char*);
```

Comment:
Removing the old API seems wrong.

Reply:
Oh! You're right, I didn't see that. I saw that you removed it entirely from the cpp file and assumed that it'll be missing here, too.

I'm not sure whether we'll run into trouble due to it being a inline function (i.e. cannot be converted to a function pointer), but I guess we'll see soon enough.

Reply:
Does the `mkstemp` symbol end up in the LibC binary like this? If it doesn't, I'd say that this is a blocker. If it does, I'd still count it as "meh" at the very least.

Reply:
I added `mkstemp` back in and just call `mkstemps` with a path length of 0 in the cpp file.

This avoids code duplication at the cost of an extra function call.

Reply:
is there something in Ports that can be used as a test case to determine if its yay/nay (last comment implies no longer an issue)

Reply:
One can just use any standard way of getting the list of exported symbols to check if `mkstemp` is present (e.g. `nm -gD libc.so`, etc.).